src/app/garfile/manifest/index.njk

```diff
@@ -26,19 +24,19 @@
   {% endif %}
 
 <div class="govuk-grid-row">
-	<div class="govuk-grid-column-two-thirds">
-		<form action="/garfile/manifest" method="post">
-			<span class="govuk-caption-xl">{{__('section_count', 4, 8)}}</span>
-			<h1 class="govuk-heading-xl">{{__('heading_manifest_details')}}</h1>
-			<h2 class="govuk-heading-l">{{ __('title_people') }}</h2>
-			<p class="govuk-body-l">{{__('caption_manifest_details')}}</p>
-  </div>
-  <div class="govuk-grid-column-full">
-        {% if savedPeople | length %}
+  <form action="/garfile/manifest" method="post">
```

Comment:
Moved the form outside as it looked wrong having it inside a div that then is closed before the form.